.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      sdk_name:
+        description: SDK to release (raven, rails, ruby, sidekiq)
+        required: true
+      version:
+        description: Version to release
+        required: true
+      force:
+        description: Force a release even when there are release-blockers (optional)
+        required: false
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: "Release a new version"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GH_RELEASE_PAT }}
+          fetch-depth: 0
+      - name: Prepare release
+        uses: getsentry/action-prepare-release@v1.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
+        with:
+          path: sentry-${{ inputs.sdk_name }}
```

Comment:
I think we can just accept `inputs.sdk_name` here. I feel more comfortable to always enter the full gem name.

Reply:
Alright, will update accordingly.